internal/cli/roles_permissions.go

```diff
@@ -56,10 +56,10 @@ func listRolePermissionsCmd(cli *cli) *cobra.Command {
 		Aliases: []string{"ls"},
 		Args:    cobra.MaximumNArgs(1),
 		Short:   "List permissions defined within a role",
-		Long: `List existing permissions defined in a role. To add a permission try:
-auth0 roles permissions add <role-id>`,
-		Example: `auth0 roles permissions list <role-id>
-auth0 roles permissions ls`,
+		Long:    "List existing permissions defined in a role. To add a permission, run: `auth0 roles permissions add`.",
+		Example: `  auth0 roles permissions list
+  auth0 roles permissions list <role-id>
+  auth0 roles permissions ls --json`,
```

Comment:
As a given role is likely to have many permissions, consider in a future PR adding an example with the `-n` flag, like in other similar commands.